scipy/stats/tests/test_morestats.py

```diff
@@ -1193,27 +1242,6 @@ def test_mood_alternative(self):
         with pytest.raises(ValueError, match="alternative must be..."):
             stats.mood(x, y, alternative='ekki-ekki')
 
-    @pytest.mark.xfail(reason="SciPy needs tie correction like R (gh-13730)")
```

Comment:
Even though `test_against_SAS` is stronger, let's leave this one in. Remove the `xfail`, update the values to the correct ones from SAS, and change the name (e.g. `test_against_SAS_2`). One good thing is that this does different `alternative`s.